test/dlc_tests/container_tests/bin/pytorch_tests/testTorchaudio

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+HOME_DIR=/test
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs
+
+S3_IO_DEV_BRANCH="v0.11.0-rc2"
```

Comment:
Why specifically this branch, rather than the version of torchaudio installed on the image?

Also, please rename the script variable names from `S3_IO*` to `TORCHAUDIO*` or something like that.